### PR TITLE
docs: add sample repository reference in ci setup 

### PR DESCRIPTION
**Pull request by Harish-osmosys**
Added Sample Repository link for a reference if any change need to be done in future we can modify the repository also 

## Summary by CodeRabbit

- **Documentation**
	- Updated CI setup guides for Angular, PHP, and Python projects on both GitHub and GitLab.
	- Reorganized and reordered sections for improved clarity and consistency in instructions.
	- Introduced a new "Sample Repository" section with practical demonstration links.
	- Added or renamed "Conclusion" sections for better guide summarization.

### CODE REVIEW

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]
Python gitlab is missing.
Can we add that too?

File: ci-setups/angular-github.md [hunk 1]
```diff
@@ -26,7 +26,8 @@
   - [7. Troubleshooting and Advanced Configuration](#7-troubleshooting-and-advanced-configuration)
     - [Handling Merge Conflicts](#handling-merge-conflicts)
     - [Customizing Scripts](#customizing-scripts)
-  - [8. Conclusion](#8-conclusion)
+  - [8. Sample Repository](#8-sample-repository)
+  - [9. Conclusion](#9-conclusion)
     - [Benefits of Workflow Setup](#benefits-of-workflow-setup)
     - [Future Enhancements](#future-enhancements)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/angular-github.md:33]: Original lines [51-51]: brackets are present, there is no broken code

Original lines [58-58]: updated folder name to `.github` as this is used by company

Original lines [78-78]: updated

Original lines [87-87]: updated

Original lines [182-182]: updated

File: ci-setups/angular-gitlab.md [hunk 2]
```diff
@@ -31,7 +31,8 @@
     - [1. Job Artifacts](#1-job-artifacts)
       - [Update your `.gitlab-ci.yml` Configuration](#update-your-gitlab-ciyml-configuration)
       - [Usage](#usage)
-  - [9. Conclusion](#9-conclusion)
+  - [9. Sample Repository](#9-sample-repository)
+  - [10. Conclusion](#10-conclusion)
     - [Benefits of CI Setup](#benefits-of-ci-setup)
     - [Future Enhancements](#future-enhancements)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/angular-gitlab.md:38]: Original lines [63-63]: It is already correct, same line is written as coderabbit suggested

Original lines [83-83]: updated

Original lines [91-91]: updated

Original lines [125-125]: It is already correct, same line is written as coderabbit suggested, however, added backticks for emphasis

Original lines [164-164]: used different sentence, updated

Original lines [179-179]: used different sentence, updated

Original lines [183-183]: It is already correct, same line is written as coderabbit suggested, however, added backticks for emphasis

Original lines [204-204]: updated

Original lines [228-228]: It is already correct, same line is written as coderabbit suggested

Original lines [237-237]: updated

Original lines [252-252]: updated

Original lines [304-304]: updated

Original lines [307-307]: updated

File: ci-setups/php-github.md [hunk 3]
```diff
@@ -28,7 +28,8 @@
 7. [Troubleshooting and Advanced Configuration](#7-troubleshooting-and-advanced-configuration)
     - [Handling Merge Conflicts](#handling-merge-conflicts)
     - [Customizing Steps](#customizing-steps)
-8. [Conclusion](#8-conclusion)
+8. [Sample Repository](#8-sample-repository)
+9. [Conclusion](#9-conclusion)
     - [Benefits of CI Setup](#benefits-of-ci-setup)
     - [Future Enhancements](#future-enhancements)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/php-github.md:35]: Original lines [52-52]: It is already correct, same line is written as coderabbit suggested

Original lines [72-72]: updated

Original lines [123-123]: Can leave as it is, makes sense in this context

Original lines [215-215]: updated

File: ci-setups/php-gitlab.md [hunk 4]
```diff
@@ -27,7 +27,8 @@
 7. [Troubleshooting and Advanced Configuration](#7-troubleshooting-and-advanced-configuration)
     - [Handling Merge Conflicts](#handling-merge-conflicts)
     - [Customizing Scripts](#customizing-scripts)
-8. [Conclusion](#8-conclusion)
+8. [Sample Repository](#8-sample-repository)
+9. [Conclusion](#9-conclusion)
     - [Benefits of CI Setup](#benefits-of-ci-setup)
     - [Future Enhancements](#future-enhancements)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/php-gitlab.md:34]: Original lines [39-39]: updated

Original lines [53-53]: updated

Original lines [63-63]: updated

Original lines [104-104]: updated

Original lines [115-115]: updated

Original lines [196-196]: updated

Original lines [202-202]: left this as is, makes sense in this context

Original lines [220-220]: reframed the sentence

File: ci-setups/python-github.md [hunk 5]
```diff
@@ -93,7 +94,13 @@ Explore advanced GitHub Actions features such as:
...
 If your CI build fails, check the logs in GitHub Actions. Ensure your local development environment matches the CI environment.
 
-## 11. Conclusion
+## 11. Sample Repository
+
+[Repository Link](https://github.com/OsmosysSoftware/angular-eslint-workflow-guide) 
+
+Explore this for practical demonstration of CI setups.
+
+## 12. Conclusion
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/python-github.md]: Updated the link

File: ci-setups/python-github.md [hunk 6]
```diff
@@ -12,7 +12,8 @@
 - [8. Example YAML Configuration](#8-example-yaml-configuration)
 - [9. Advanced Features](#9-advanced-features)
 - [10. Troubleshooting](#10-troubleshooting)
-- [11. Conclusion](#11-conclusion)
+- [11. Sample Repository](#11-sample-repository)
+- [12. Conclusion](#12-conclusion)
 
 ## 1. Introduction
 This document provides guidelines for setting up Continuous Integration and Continuous Deployment (CI/CD) for Python projects on GitHub using GitHub Actions. It aims to ensure consistent and reliable workflows for Python development.
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/python-github.md:19]: Original lines [25-25]: no changes needed

Original lines [36-36]: it already has "the" article

Original lines [40-40]: no need to change, makes sense in this context

Original lines [108-108]: did not change, makes sense in this context

File: ci-setups/python-github.md [hunk 7]
```diff
@@ -12,7 +12,8 @@
 - [8. Example YAML Configuration](#8-example-yaml-configuration)
 - [9. Advanced Features](#9-advanced-features)
 - [10. Troubleshooting](#10-troubleshooting)
-- [11. Conclusion](#11-conclusion)
+- [11. Sample Repository](#11-sample-repository)
+- [12. Conclusion](#12-conclusion)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/python-github.md:16]: No need, these are correctly formatted for table of contents

File: ci-setups/python-gitlab.md [hunk 8]
```diff
@@ -18,7 +18,8 @@
     - [Writing Jobs for Linting and Testing](#writing-jobs-for-linting-and-testing)
     - [Adding Workflow Rules](#adding-workflow-rules)
     - [Complete `.gitlab-ci.yml` Configuration](#complete-gitlab-ciyml-configuration)
-  - [4. Conclusion](#4-conclusion)
+  - [4. Sample Repository](#4-sample-repository)
+  - [5. Conclusion](#5-conclusion)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/python-gitlab.md]: No need, these are correctly formatted for table of contents

File: ci-setups/angular-github.md [hunk 9]
```diff
@@ -26,7 +26,8 @@
   - [7. Troubleshooting and Advanced Configuration](#7-troubleshooting-and-advanced-configuration)
     - [Handling Merge Conflicts](#handling-merge-conflicts)
     - [Customizing Scripts](#customizing-scripts)
-  - [8. Conclusion](#8-conclusion)
+  - [8. Sample Repository](#8-sample-repository)
+  - [9. Conclusion](#9-conclusion)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/angular-github.md:30]: No need, these are correctly formatted for table of contents

File: ci-setups/php-github.md [hunk 10]
```diff
@@ -28,7 +28,8 @@
 7. [Troubleshooting and Advanced Configuration](#7-troubleshooting-and-advanced-configuration)
     - [Handling Merge Conflicts](#handling-merge-conflicts)
     - [Customizing Steps](#customizing-steps)
-8. [Conclusion](#8-conclusion)
+8. [Sample Repository](#8-sample-repository)
+9. [Conclusion](#9-conclusion)
```
[the remark this answers is not in the document]
Comment (kshitij-k-osmosys) [reply ci-setups/php-github.md:32]: No need, these are correctly formatted for table of contents

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**